torch/csrc/autograd/record_function.h

```diff
@@ -121,8 +121,11 @@ struct TORCH_API RecordFunction {
     return threadId_;
   }
 
- private:
+ protected:
   void processCallbacks();
+  // Runs the end callbacks that were pushed to the callback manager. Throws if
+  // the current RecordFunction is not initialized.
+  void runEndCallbacks();
```

Comment:
Looks like this function and `processCallbacks` works in as a pair. If this is true, should we name them accordingly, e.g., `runStartCallbacks()` and `runEndCallbacks()`?

Reply:
Yes, I think it should be more aptly named to runStartCallbacks. It only invokes the starting ones